asteroid/masknn/blocks.py

```diff
@@ -50,6 +50,50 @@ def forward(self, x):
         return res_out, skip_out
 
 
+class SeparableDilatedConv2DBlock(nn.Module):
```

Comment:
I'd rather have one `Conv1DBlock` which supports with and without skip connections. 
And have `TDConvNet` support this argument as well.
Does this sound feasible?

Reply:
As requested and discussed I will take it away from your core codebase.